web-client/src/presenter/computeds/dashboardExternalHelper.test.js

```diff
@@ -36,50 +36,65 @@ describe('petitioner dashboard helper', () => {
     });
     expect(result.showCaseList).toEqual(true);
     expect(result.showWhatToExpect).toEqual(false);
-    expect(result.showCaseSearch).toEqual(false);
   });
-  it('shows case search if defined user has privatePractitioner role', () => {
+
+  it('should keep the showFileACase flag as false when the user role is not a private practitioner', () => {
+    applicationContext.getCurrentUser = () => ({
+      role: ROLES.petitioner,
+    });
+
+    const result = runCompute(dashboardExternalHelper, {
+      state: {
+        closedCases: [{ something: true }],
+        openCases: [{ something: true }],
+      },
+    });
+
+    expect(result.showFileACase).toEqual(false);
+  });
+
+  it('should set the showFileACase flag as true when the user role is a private practitioner', () => {
     applicationContext.getCurrentUser = () => ({
       role: ROLES.privatePractitioner,
     });
+
     const result = runCompute(dashboardExternalHelper, {
       state: {
         closedCases: [{ something: true }],
         openCases: [{ something: true }],
       },
     });
-    expect(result.showCaseList).toEqual(true);
-    expect(result.showWhatToExpect).toEqual(false);
-    expect(result.showCaseSearch).toEqual(true);
+
+    expect(result.showFileACase).toEqual(true);
   });
 
-  it('shows case search if defined user has irsPractitioner role', () => {
+  it('should keep the showStartButton flag as false when the user role is not a private practitioner or petitioner', () => {
```

Comment:
Perhaps to keep things less confusing, explicitly state `irsPractitioner`

Reply:
done